include/core/mediator/UnitsHelper.hpp

```diff
@@ -25,8 +25,11 @@ class UnitsHelper {
         {
             throw std::runtime_error("Unable to create UDUNITS2 Unit System.");
         }
+        #ifndef UDUNITS_QUIET
```

Comment:
Not required change, but should this just be always included? Turn off upstream error messages, we will report ourselves? Seems like this should only be there in a loglevel=DEBUG type world.

Reply:
I'm OK with doing that.  That is somewhat why I used different `QUIET` options/names, in case we ever wanted to change them up independently.